src/main/java/frc/robot/commands/AutoVisionCommand.java

```diff
@@ -0,0 +1,78 @@
+package frc.robot.commands;
+
+import com.pathplanner.lib.path.GoalEndState;
+import com.pathplanner.lib.path.PathConstraints;
+import com.pathplanner.lib.path.PathPlannerPath;
+
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Transform2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants.limelightCamMode;
+import frc.robot.Constants.limelightLedMode;
+import frc.robot.Constants.limelightPipeline;
+import frc.robot.subsystems.conveyor.Conveyor;
+import frc.robot.subsystems.drive.DriveSubsystem;
+import frc.robot.subsystems.elevator.Elevator;
+import frc.robot.subsystems.intake.Intake;
+import frc.robot.subsystems.intakeLimelight.IntakeLimelight;
+import frc.robot.subsystems.lights.Lights;
+import frc.robot.subsystems.shooter.Shooter;
+import java.util.List;
+import java.util.Optional;
+
+public class AutoVisionCommand extends SequentialCommandGroup {
+  public Optional<PathPlannerPath> path = Optional.empty();
+  public Optional<Pair<Pose2d, Pose2d>> poses = Optional.empty();
+  public AutoVisionCommand(
+      DriveSubsystem drive,
+      Intake intake,
+      Conveyor conveyor,
+      Elevator elevator,
+      Shooter shooter,
+      Lights lights,
+      IntakeLimelight intakeLimelight) {
+    addRequirements(drive, intake, conveyor, elevator);
```

Comment:
i don't think so

Reply:
Pretty sure it should, because this command actually uses the shooter as far as I can tell. I don't remember how requirement inheritance works (i.e. `IntakeSequence` requires the shooter, so I *think* this will automatically require it?) but it's probably best practice to explicitly state it at the top – if for no reason other than clarity for the reader. 

~~it would be really embarrassing for me if i was wrong~~